frag-nav/src/test/java/com/ncapdevi/fragnav/FragNavControllerRaceConditionSpec.kt

```diff
@@ -168,7 +168,7 @@ class FakeFragmentTransaction(private val parent: FakeFragmentManager) {
 
     fun create(): FragmentTransaction {
         return mock {
-            on { add(any(), any(), any()) } doAnswer { invocationOnMock ->
+            on { add(any<Int>(), any<Fragment>(), any<String>()) } doAnswer { invocationOnMock ->
```

Comment:
the new Fragment library version 1.2.5 has another `.add()` method with 3 parameters. We need to be explicit on which one we want to mock.